zipline/gens/brokers/ib_broker.py

```diff
@@ -807,14 +833,14 @@ def _update_from_order_status(zp_order, ib_order_id):
                 open_order_state = self._tws.open_orders[ib_order_id]['state']
 
                 zp_status = self._ib_to_zp_status(open_order_state.m_status)
-                if zp_status:
-                    zp_order.status = zp_status
-                else:
+                if zp_status is None:
```

Comment:
Not sure why is this changed

Reply:
from your TAO branch, just took it over

Reply:
Please cherry-pick instead of copy paste. thanks.

Reply:
hi guys, there's an additional check here a few lines below line 836 that also needs the same fix (because of the use of custom enum()s they start at 0 and thus evaluate to 0)